.pre-commit-config.yaml

```diff
@@ -34,22 +34,17 @@ repos:
       - id: backend-openapi
         name: backend openapi
         entry: pdm run -p backend/ generate_openapi --file openapi-schema.yml
-        language: python
+        language: system
         files: backend/.*
         pass_filenames: false
 
   # frontend checks
-  - repo: https://github.com/pre-commit/mirrors-eslint
-    rev: v8.35.0
+  - repo: local
     hooks:
       - id: eslint
-        additional_dependencies:
-          - eslint@8.35.0
-          - eslint-config-prettier@8.6.0
-          - eslint-plugin-import@2.27.5
-          - eslint-plugin-react@7.32.2
-          - "@typescript-eslint/eslint-plugin@5.54.0"
-          - "@typescript-eslint/parser@5.54.0"
+        entry: eslint
```

Comment:
Can we add this into one of the pnpm commands and then also manage the eslint version via the package.json?